README.md

```diff
@@ -45,3 +45,120 @@ Once the above variables are set and the /payload directory has been populated,
 hack/run-ci-e2e-test.sh
 ```
 
+## Bundling the Windows Machine Config Operator
+This directory contains resources related to installing the WMCO onto a cluster using OLM.
+
+### Pre-requisites
+[operator-sdk](https://github.com/operator-framework/operator-sdk) and
+[opm](https://github.com/operator-framework/operator-registry/) have been installed on the localhost.
+
+### Generating a new bundle
+If changes need to be made to the bundle spec, a new bundle can be generated with:
```

Comment:
This leads me to ask what do I need to do if I am starting from scratch. Do I not need to run this step?

Reply:
You wont be starting from "scratch", as the bundle i've seeded here now exists. We just need to update the bundle with the generate command in the case that we change the yaml files. I've updated the PR to explain that.

I've also created https://issues.redhat.com/browse/WINC-332 to ensure that the bundle is being updated when necessary.